Gems/PhysX/Code/Source/EditorHingeJointComponent.cpp

```diff
@@ -100,23 +101,19 @@ namespace PhysX
 
     float EditorHingeJointComponent::GetLinearValue(const AZStd::string& parameterName)
     {
-        if (parameterName == PhysX::EditorJointComponentMode::s_parameterMaxForce)
+        if (parameterName == PhysX::JointsComponentModeCommon::ParamaterNames::MaxForce)
```

Comment:
question: Would this function be better implemented as a switch?

Reply:
these are strings, so it wouldn't work.
i do want to come back and remove the need for string usage here though.

Reply:
Aaah I see... bummer - okay nice one makes sense (but not in this PR) Cheers!